pkg/components/networking/networkobservability/component.go

```diff
@@ -9,11 +9,12 @@ type Component struct {
 	*config.Component
 }
 
-var NetworkToolsComponent = Component{
+var NetworkObservabilityComponent = Component{
 	Component: &config.Component{
-		Name:                 "Networking / network-tools",
+		Name:                 "Networking / Network Observability",
 		Operators:            []string{},
-		DefaultJiraComponent: "Networking / network-tools",
+		DefaultJiraProject:   "NETOBSERV",
```

Comment:
@stbenjam - I added `DefaultJiraProject` to be our project name , hope that's okay.

Reply:
It's not, the project and component are used together, i.e. the Network Observability component under OCPBUGS.

Leave this blank so it defaults to OCPBUGS. I don't think alternative projects are supported yet.